brage-import/src/main/java/no/sikt/nva/brage/migration/mapper/BrageNvaMapper.java

```diff
@@ -163,12 +166,27 @@ private static AssociatedLink extractAssociatedLink(Record brageRecord) {
     }
 
     private static Set<AdditionalIdentifierBase> extractAdditionalIdentifiers(Record brageRecord) {
-        return Stream.of(extractCristinAdditionalIdentifier(brageRecord), extractBrageHandle(brageRecord))
+        return Stream.of(extractCristinAdditionalIdentifier(brageRecord),
+                         extractBrageHandle(brageRecord),
+                         extractInsperaIdentifier(brageRecord),
+                         extractWiseflowIdentifier(brageRecord))
                    .filter(Optional::isPresent)
                    .map(Optional::get)
                    .collect(Collectors.toSet());
     }
 
+    private static Optional<? extends AdditionalIdentifierBase> extractInsperaIdentifier(Record brageRecord) {
+        return isNull(brageRecord.getInsperaIdentifier())
+                   ? Optional.empty()
+                   : Optional.of(new AdditionalIdentifier(INSPERA_SOURCE, brageRecord.getInsperaIdentifier()));
```

Comment:
Should we create dedicated AdditionalIdentifier types for this? especially if this is new data in our system, I think it would be a good idea, if this already exists in prod its probably ok

Reply:
See other PR…I imagine these values are URNs

Reply:
I am thinking about keeping it as AdditionalIdentifier for now, because instance SourceName.instanceName for this identifiers does not make much sense, cause it will be missing when importing from Inspera and Wiseflow directly, and refactoring to serialize SourceName as object requires migration in test.